src/test/java/com/google/api/services/bigquery/MinifiedBigqueryTest.java

```diff
@@ -0,0 +1,34 @@
+package com.google.api.services.bigquery;
+
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.testing.http.MockHttpTransport;
+import com.google.api.client.testing.json.MockJsonFactory;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+public class MinifiedBigqueryTest {
```

Comment:
@stalbot followup from previous PR.
Doesn't test return but at least validates we send the request param.